abichecker/abichecker.py

```diff
@@ -883,7 +883,7 @@ def ensure_settled(self, src_project, src_srcinfo, matchrepos):
                 repository = [ mr.srcrepo for mr in matchrepos],
                 arch = [ mr.arch for mr in matchrepos])
         for result in results:
-            for res in osc.core.result_xml_to_dicts(result):
+            for res, _ in osc.core.result_xml_to_dicts(result):
```

Comment:
Curious, the last time this was [changed according to blame was 2 years ago](https://github.com/openSUSE/osc/blame/f12c7747d0697a718e20a698a1816a20e299d897/osc/core.py#L5637).

Reply:
the bot ran on a local git checkout for ages and maintenance didn't bother about it

Reply:
actually it uses the package but a very old one